Sources/Bow/Effects/IO.swift

```diff
@@ -351,7 +351,11 @@ public class IOMonad : IOApplicative, Monad {
     }
 }
 
-public class IOAsyncContext : AsyncContext {
+public class IOAsync : IOMonadError, Async {
+    public func deferExecution<A>(_ fa: @escaping () -> Kind<ForIO, A>) -> Kind<ForIO, A> {
```

Comment:
unless it collides with existing keyword this operation is frequently named `suspend` and it's variation that automatically calls `pure` or `raiseError` is frequently called `delay`